app/assets/javascripts/application.js

```diff
@@ -1,3 +1,5 @@
 //= require govuk_publishing_components/dependencies
 //= require govuk_publishing_components/lib
 //= require govuk_publishing_components/components/copy-to-clipboard
+
+//= require components/autocomplete
```

Comment:
Overall approach looks good so far - I'll hold off on a full review until the POST logic is implemented.

But while I'm here, there are two things I'm not keen on with this component import work:

1. That we're copying and pasting code rather than reusing a module
2. That the existing component is considered inaccessible and [ripe for replacement](https://trello.com/c/zLNLSKok/85-design-accessible-replacement-for-accessible-autocomplete-multiselect)

We could address 1 fairly easily, by publishing accessible-autocomplete-multiselect to NPM and pulling from there. I think we should at least do that IMO (we may decide to invest time in improving the component rather than replacing it, in which case we should have our improvements apply to both publishing apps).

But I think we should also discuss 2 and acknowledge in the commit message that it has issues but that it's our only option at the moment?

Reply:
UPDATE after dev catchup today:

1. Let's continue with this as-is (you're right that the main code for this is already NPM, this is just 'scaffold' around it. Would be nice to put this into Rubygems but you point out we don't have a precedent for individual components)
2. We'll look at prioritising bringing https://trello.com/c/zLNLSKok/85-design-accessible-replacement-for-accessible-autocomplete-multiselect into a sprint in the near future, to replace this component.